python310Packages.django-reversion: 5.0.8 -> 5.0.10

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python310Packages.django-reversion is: An extension to the Django web framework that provides comprehensive version control facilities

meta.homepage for python310Packages.django-reversion is: https://github.com/etianen/django-reversion

meta.changelog for python310Packages.django-reversion is: https://github.com/etianen/django-reversion/blob/v5.0.9/CHANGELOG.rst


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.0.9 with grep in /nix/store/bfa1znfa5dy6k86qkhw30yb77z6kpgv7-python3.10-django-reversion-5.0.9
- found 5.0.9 in filename of file in /nix/store/bfa1znfa5dy6k86qkhw30yb77z6kpgv7-python3.10-django-reversion-5.0.9

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2 total rebuild path(s)

2 package rebuild(s)

First fifty rebuilds by attrpath
python310Packages.django-reversion
python311Packages.django-reversion
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/bfa1znfa5dy6k86qkhw30yb77z6kpgv7-python3.10-django-reversion-5.0.9 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python310Packages.django-reversion https://github.com/r-ryantm/nixpkgs/archive/eaeaeffc5171bc259ab3371d85a1c33e3686620a.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/eaeaeffc5171bc259ab3371d85a1c33e3686620a#python310Packages.django-reversion
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/bfa1znfa5dy6k86qkhw30yb77z6kpgv7-python3.10-django-reversion-5.0.9
ls -la /nix/store/bfa1znfa5dy6k86qkhw30yb77z6kpgv7-python3.10-django-reversion-5.0.9/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.django-reversion</li>
    <li>python311Packages.django-reversion</li>
  </ul>
</details>

---

###### Maintainer pings

r-ryantm committed:
python310Packages.django-reversion: 5.0.8 -> 5.0.9

natsukium commented:
It seems that the tag is wrong.
https://github.com/etianen/django-reversion/commit/6474b8ab52a663cf70e4a60edeaea3ff5fbbe5c0#commitcomment-135826421

natsukium committed:
python311Packages.django-reversion: switch to pypa build

natsukium committed:
python311Packages.django-reversion: 5.0.9 -> 5.0.10

Changelog: https://github.com/etianen/django-reversion/blob/v5.0.10/CHANGELOG.rst

natsukium commented:
Result of `nixpkgs-review pr 277164` at 1c51b3ca run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python310Packages.django-reversion</li>
    <li>python311Packages.django-reversion</li>
  </ul>
</details>
